backend/src/serverless/integrations/services/integrations/premium/linkedinIntegrationService.ts

```diff
@@ -26,7 +26,7 @@ import { getAllPostReactions } from '../../../usecases/linkedin/getPostReactions
 
 export class LinkedinIntegrationService extends IntegrationServiceBase {
   constructor() {
-    super(IntegrationType.LINKEDIN, 20)
+    super(IntegrationType.LINKEDIN, 60)
```

Comment:
Does this mean that the refresh period will now be 60mins?
I'm asking because I'll probably need to update https://docs.crowd.dev/docs/linkedin-integration